openscap_daemon/oscap_helpers.py

```diff
@@ -113,10 +124,18 @@ def scrape_profiles(tree, namespace, dest):
         return ret
 
     scrape_profiles(
-        input_tree, "http://checklists.nist.gov/xccdf/1.1", ret
+        input_tree,
+        xccdf_id,
+        "http://checklists.nist.gov/xccdf/1.1",
+        "http://scap.nist.gov/schema/scap/source/1.1",
+        ret
     )
     scrape_profiles(
-        input_tree, "http://checklists.nist.gov/xccdf/1.2", ret
+        input_tree,
+        xccdf_id,
+        "http://scap.nist.gov/schema/scap/source/1.2",
+        "http://checklists.nist.gov/xccdf/1.2",
```

Comment:
Why is this namespace second here and first in the previous call?

Reply:
It should be the same! I will correct it.